lib/AuroraLip/Archives/Formats/PCKG_CING.cs

```diff
@@ -0,0 +1,57 @@
+﻿using AuroraLip.Common;
+using System;
+using System.IO;
+
+namespace AuroraLip.Archives.Formats
+{
+    class PCKG_CING : Archive, IMagicIdentify
+    {
+        public bool CanRead => true;
+
+        public bool CanWrite => false;
+
+        public string Magic => magic;
+
+        private const string magic = "PCKG";
+
+        public PCKG_CING() { }
+
+        public PCKG_CING(string filename) : base(filename) { }
+
+        public PCKG_CING(Stream stream, string filename = null) : base(stream, filename) { }
+
```

Comment:
public bool IsMatch(Stream stream, in string extension = "")
            => stream.MatchString(magic);